includes/burstSafetyModeFunctions.php

```diff
@@ -0,0 +1,44 @@
+<?php 
+namespace NewfoldLabs\WP\Module\Performance;
+use NewfoldLabs\WP\Module\Performance\CacheTypes\Browser;
+use NewfoldLabs\WP\Module\Performance\CacheTypes\Skip404;
+use NewfoldLabs\WP\Module\Performance\ResponseHeaderManager;
+
+$newfold_burst_safety_mode = (bool) get_option( 'newfold_burst_safety_mode' );
+
+$site_cache_level = get_option( 'newfold_cache_level', 3 );
+$site_skip404 = (bool) get_option( 'newfold_skip_404_handling', true );
+
+if ( $newfold_burst_safety_mode === false ) {
+
+}else{
+	
+}
```

Comment:
Shall we remove the empty `if-else` statement?

Reply:
yes, I forgot remove block